{"status": "broken", "description": "Obstacle placement for the navigation test.\n        ", "labels": [{"name": "tag", "value": "hypothesis"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "nav_test"}, {"name": "subSuite", "value": "TestNavigation"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "31495-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.nav_test"}], "stop": 1606101044799, "uuid": "caf693a8-11ee-405c-bb00-2d1c62b2148c", "historyId": "09b3e2b6796b6a978f8f914c17f56fd7", "name": "test_obstacle_placement", "start": 1606101043945, "fullName": "tests.nav_test.TestNavigation#test_obstacle_placement", "statusDetails": {"message": "Flaky: Hypothesis test_obstacle_placement(self=<tests.nav_test.TestNavigation instance at 0x7f8c67f88e60>, obstacle='minicoffeetable') produces unreliable results: Falsified on the first call but did not on a subsequent one", "trace": "self = <tests.nav_test.TestNavigation instance at 0x7f8c67f88e60>\n\n    @settings(max_examples=1)\n>   @given(st.sampled_from(['minicoffeetable','coffeetable']))\n    def test_obstacle_placement(self,obstacle):\n        \"\"\"Obstacle placement for the navigation test.\n\ntests/nav_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python2.7/dist-packages/hypothesis/core.py:656: in execute_once\n    % (test.__name__, text_repr[0])\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <hypothesis.core.StateForActualGivenExecution object at 0x7f8c71184b10>\nmessage = \"Hypothesis test_obstacle_placement(self=<tests.nav_test.TestNavigation instance at 0x7f8c67f88e60>, obstacle='minicoffeetable') produces unreliable results: Falsified on the first call but did not on a subsequent one\"\n\n    def __flaky(self, message):\n        if len(self.falsifying_examples) <= 1:\n>           raise Flaky(message)\nE           Flaky: Hypothesis test_obstacle_placement(self=<tests.nav_test.TestNavigation instance at 0x7f8c67f88e60>, obstacle='minicoffeetable') produces unreliable results: Falsified on the first call but did not on a subsequent one\n\n/usr/local/lib/python2.7/dist-packages/hypothesis/core.py:856: Flaky"}, "attachments": [{"source": "a05ed062-e2ab-4f8b-84b5-02b083b134d7-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "e5138147-430c-4ce5-afc7-57cf7748739a-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "860ca45b-858a-4d5b-b9d3-1ec406c65f66-attachment.txt", "type": "text/plain", "name": "stderr"}]}